src/main/bash/sdkman-install.sh

```diff
@@ -193,6 +194,12 @@ function __sdkman_checksum_zip() {
 	local algorithm checksum cmd
 	local shasum_avail=false
 	local md5sum_avail=false
+	
+	if [ ! -f "${headers_archive}" ]; then
```

Comment:
I must have overlooked this variable name in the past, but it's a bit confusing to me. It isn't pointing to a headers archive but a headers file. Would you mind renaming it for the sake of clarity?

Reply:
s/headers*/headers_file//